assets/src/components/content/add_resource_content/AddOther.tsx

```diff
@@ -10,17 +10,19 @@ export const AddOther: React.FC<Props> = ({ onAddItem, index }) => {
   return (
     <>
       <div className="list-group">
-        <a
-          href="#"
+        <button
           key={'static_activity_bank'}
           className="list-group-item list-group-item-action flex-column align-items-start"
-          onClick={() => onAddItem(createDefaultSelection(), index)}
+          onClick={() => {
+            onAddItem(createDefaultSelection(), index);
+            document.body.click();
```

Comment:
react-bootstrap popover closes on "click outside," so force a click if a content item is added.